apps/E2E/src/common/BasePage.ts

```diff
@@ -163,7 +163,17 @@ export abstract class BasePage {
       }
       default:
       case MobilePlatform.Android:
-        // Todo
+        await browser.waitUntil(
+          async () => {
+            const buttonElementSelector = `new UiScrollable(new UiSelector().description("${TESTPAGE_BUTTONS_SCROLLVIEWER}").scrollable(true)).setMaxSearchSwipes(10).setAsVerticalList().scrollIntoView(new UiSelector().description("${this._pageButtonName}"))`;
```

Comment:
Can we add a comment about this selector? Why can't we use the `By()` function here? What type of selector is this?